i2c/mcp9808_i2c/mcp9808_i2c.c

```diff
@@ -1,7 +1,7 @@
 /**
  * Copyright (c) 2020 Raspberry Pi (Trading) Ltd.
  *
- * SPDX-License-Identifier: BSD-3-Clause
+ * SPDx-License-Identifier: BSD-3-Clause

```

Comment:
Doh, the `SPDX` should remain in uppercase (I guess you must have search'n'replaced `X` to `x`?)

Reply:
Aha yes, fixed up